app/assets/v2/images/chains/binance.svg

```diff
@@ -0,0 +1,24 @@
+        <path id="Path_88" data-name="Path 88" d="M154.609,82.187c0-1.039.928-1.857,2.52-1.857a7.028,7.028,0,0,1,4.2,1.592l1.459-2.056a8.577,8.577,0,0,0-5.615-1.945c-3.073,0-5.283,1.813-5.283,4.51,0,2.874,1.857,3.846,5.151,4.642,2.874.663,3.493,1.26,3.493,2.387,0,1.194-1.061,1.967-2.741,1.967a7.126,7.126,0,0,1-4.885-2.012l-1.636,1.945a9.532,9.532,0,0,0,6.455,2.476c3.25,0,5.527-1.724,5.527-4.642,0-2.586-1.7-3.758-4.974-4.554C155.316,83.933,154.609,83.4,154.609,82.187Z" transform="translate(-81.604 -42.034)" fill="#f0b90b"/>
+        <path id="Path_89" data-name="Path 89" d="M195.76,82.777v11.1h2.719V78.4h-2.9l-4.709,7.317L186.166,78.4h-2.9V93.874h2.675V82.821l4.841,7.251h.088Z" transform="translate(-98.866 -42.293)" fill="#f0b90b"/>
+        <path id="Path_90" data-name="Path 90" d="M230.431,78.16h-2.52L221.1,93.745h2.785l1.592-3.736H232.8l1.57,3.736h2.874ZM231.8,87.6h-5.328l2.653-6.19Z" transform="translate(-119.274 -42.164)" fill="#f0b90b"/>
+        <path id="Path_91" data-name="Path 91" d="M274.074,93.874,269.9,88.016a4.634,4.634,0,0,0,3.692-4.709c0-3.029-2.189-4.908-5.748-4.908h-6.9V93.874h2.719V88.481h3.4l3.8,5.394Zm-3.25-10.412c0,1.592-1.238,2.609-3.183,2.609h-3.979V80.876h3.957C269.631,80.876,270.824,81.76,270.824,83.463Z" transform="translate(-140.767 -42.293)" fill="#f0b90b"/>
+        <path id="Path_92" data-name="Path 92" d="M299.779,80.92h4.908V78.4H292.13v2.52h4.908V93.874h2.741Z" transform="translate(-157.591 -42.293)" fill="#f0b90b"/>
+        <path id="Path_93" data-name="Path 93" d="M350.786,91.088l-1.746-1.769a6.066,6.066,0,0,1-4.443,2.012,5.52,5.52,0,0,1,0-11.009,6.154,6.154,0,0,1,4.333,1.923l1.746-2.012a8.066,8.066,0,0,0-6.057-2.41,8,8,0,0,0-.111,16A7.984,7.984,0,0,0,350.786,91.088Z" transform="translate(-181.601 -41.983)" fill="#f0b90b"/>
+        <path id="Path_94" data-name="Path 94" d="M382.706,87.353v6.521h2.719V78.4h-2.719v6.433h-7.383V78.4H372.6V93.874h2.719V87.353Z" transform="translate(-201.002 -42.293)" fill="#f0b90b"/>
+        <path id="Path_95" data-name="Path 95" d="M414.561,78.16h-2.52l-6.809,15.585h2.785l1.592-3.736h7.317l1.57,3.736h2.874Zm1.371,9.439H410.6l2.653-6.19Z" transform="translate(-218.604 -42.164)" fill="#f0b90b"/>
+        <path id="Path_96" data-name="Path 96" d="M445.41,78.4V93.874h2.719V78.4Z" transform="translate(-240.278 -42.293)" fill="#f0b90b"/>
+        <path id="Path_97" data-name="Path 97" d="M470.158,89.1l-8.29-10.7h-2.52V93.874h2.675V82.866l8.533,11.009h2.277V78.4h-2.675Z" transform="translate(-247.797 -42.293)" fill="#f0b90b"/>
+      </g>
+    </g>
+  </g>
+</svg>
```

Comment:
can we run svgo on this to compress this ?

Reply:
done!